lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +141,14 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($value instanceof OCI8Descriptor) {
```

Comment:
Meh I am out of ideas. Maybe @cjbj can clarify why it was possible before PHP7 to bind values to `SQLT_CHR` and `CLOB` type columns. It's strange that was possible before and isn't anymore now...